python/rascaline/rascaline/utils/splines.py

```diff
@@ -0,0 +1,396 @@
+"""
+Splined radial integrals
+========================
+
+Classes for generating spline points which can be used as input for the for the radial
+integral hyper paramater option. For an complete example for the LE how to use the
+classes see :ref:`example-splines`.
+
+.. autoclass:: rascaline.utils.SplinerBase
+    :members:
+    :show-inheritance:
+
+.. autoclass:: rascaline.utils.SplinerFromFunction
+    :members:
+    :show-inheritance:
+"""
+
+from abc import ABC, abstractmethod
+from typing import Callable, Dict, Optional, Union
+
+import numpy as np
+
+
+class SplinerBase(ABC):
+    """Base class for splining arbitrary radial integrals.
+
```

Comment:
Could this get a seealso pointer to the corresponding example?

Reply:
I think for the example it is not so interesting here. We can do it for the `RadialIntegralFromFunction`?

Reply:
that works!